docs/getting-started.rst

```diff
@@ -133,9 +133,9 @@ get accurate results. This should be provided with the catalog by the catalog's
 .. code-block:: python
 
     gaia = lsdb.read_hats(
-        'https://data.lsdb.io/unstable/gaia_dr3/gaia/',
+        'https://data.lsdb.io/hats/gaia/gaia/',
```

Comment:
Let us keep gaia_dr3. This is also consistent with having names for data releases for ZTF and hopefully other catalogs.

Reply:
This will be addressed with https://github.com/astronomy-commons/data.lsdb.io/issues/32.